client/relay-chain-rpc-interface/src/reconnecting_ws_client.rs

```diff
@@ -325,6 +338,8 @@ impl ReconnectingWebsocketWorker {
 	async fn new(
 		urls: Vec<Url>,
 	) -> (ReconnectingWebsocketWorker, TokioSender<RpcDispatcherMessage>) {
+		let urls = urls.into_iter().filter_map(url_to_string_with_port).collect();
```

Comment:
This silently drops urls without a port.

Reply:
I mean either the url has a user-provided port or we use the default. If the URL has no host, it fails at initial parsing already. So this should never filter something out. I will add a log message in that case however, since filtering out anything would be undesired (but passing it on is also undesired since jsonrpsee will also not take it without a port).